### PR TITLE
Dbaas 1242- Remove the observability connection configuration

### PR DESCRIPTION
**Pull request by redhatHameed**
## Description

Remove the observability configuration from the installation and subscription files.
Code deletion/refactor not required as [it checks](https://github.com/RHEcosystemAppEng/dbaas-operator/blob/main/controllers/reconcilers/observability/observability_operator_reconciler.go#L243) the configuration before creating remote write spec 

Jira : https://issues.redhat.com/browse/DBAAS-1242

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist (Definition of Done)

- [ ] All acceptance criteria specified in JIRA or in issue number have been completed
- [ ] Unit tests added that prove the fix is effective or the feature works 
- [ ] Documentation added for the feature
- [ ] all relevant tests are passing
- [ ] Code Review completed
- [ ] Verified independently by reviewer

### CODE REVIEW

File: docs/observability-operator-config.md [hunk 1]
```diff
@@ -1,10 +1,9 @@
 
 ## How do we integrate RHODA Operator with [Observability Operator](https://github.com/rhobs/observability-operator)
 
-Observability Operator will be installed similar way as we install the other provider's operators/components. The Observability operator will be responsible to install the Prometheus, using the operator-owned deployment in the same namespace where the user installs the RHODA operator.
```
Comment (jianrongzhang89) [inline docs/observability-operator-config.md:4]: The section "Apply the Secret with token, substituted in" still has references to env variables like "ADDON_NAME". Is this still valid?

Comment (redhatHameed) [reply docs/observability-operator-config.md:4]: really don't required just keeping this reference document, just incase if it required to enable again

Comment (jianrongzhang89) [reply docs/observability-operator-config.md:4]: OK

**Review by jianrongzhang89 — APPROVED** [review]
LGTM

**Pull request merged**